src/lib/definitions.js

```diff
@@ -1072,6 +1073,15 @@ function getFHPaddingEntries(index)
             define('(RP_0_S + self)[11]', DOMWINDOW),
             define('(RP_3_WA + self)[11]', OBJECT_W_SELF),
             define('(self + RP_4_A).at("-11")', ANY_WINDOW, AT),
+            define('([].concat.call()[0] + RP_4_A).at("-11")', ANY_WINDOW, AT, CALL_ON_GLOBAL),
+            define
+            (
+                '([].concat.call()[0] + RP_4_A)[SLICE_OR_SUBSTR]("-11")[0]',
+                ANY_WINDOW,
+                CALL_ON_GLOBAL
+            ),
+            define('(RP_0_S + [].concat.call()[0])[11]', CALL_ON_GLOBAL, DOMWINDOW),
+            define('(RP_3_WA + [].concat.call()[0])[11]', CALL_ON_GLOBAL, WINDOW),
```

Comment:
Instead of these definitions we could add just one new definition for `self`:

https://github.com/fasttime/JScrewIt/blob/8f233cbb59108d2cf00a88543632b47d03b420c1/src/lib/definitions.js#L1410-L1413

I think `define('[].concat.call()[0]', CALL_ON_GLOBAL, SELF_OBJ),` would cover all cases.

Reply:
why `SELF_OBJ`, not `ANY_WINDOW`?